docker-compose.yml

```diff
@@ -15,6 +15,8 @@ services:
     tty: true
     restart: on-failure
     command: ["/app/utils/init-db.sh"]
+    environment:
+      - DATABASE_URL=postgresql+psycopg2://root:root@db/perfectfit
```

Comment:
So is this something that Heroku will automatically set if we use the managed database?